node/coinstacks/ethereum/api/src/models.ts

```diff
@@ -53,35 +46,7 @@ export interface EthereumAccount extends Account {
 }
 
 /**
- * Contains ethereum specific transaction info as returned from the node
- */
-export interface EthereumTxSpecific {
-  tx: {
-    nonce: string
-    gasPrice: string
-    gas: string
-    to: string
-    value: string
-    input: string
-    hash: string
-    blockNumber: string
-    blockHash?: string
-    from: string
-    transactionIndex: string
-  }
-  receipt?: {
-    gasUsed: string
-    status: string
-    logs: Array<{
-      address: string
-      topics: Array<string>
-      data: string
-    }>
-  }
-}
-
-/**
- * Contains info about a EthereumTx
+ * Contains Ethereum specific transaction info as returned from the node
```

Comment:
This is the old comment, its not exactly whats returned from the node anymore

Reply:
I figured it's just Ethereum specific info on top of the usual Tx. Should I return "as returned from the node" part?

Reply:
Updated the comments